include/Model/Default_Languages_Proxy.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\Model;
+
+use PLL_Language;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Default proxy the list of languages.
+ *
+ * @since 3.8
+ */
+class Default_Languages_Proxy extends Abstract_Languages_Proxy {
```

Comment:
Proposing a better name here, since `Default_Languages_Proxy` suggests it does something or it manages default language.
```suggestion
class Pass_Through_Proxy extends Abstract_Languages_Proxy {
```
or
```suggestion
class No_Op_Proxy extends Abstract_Languages_Proxy {
```